public/app/features/browse-dashboards/components/BrowseView.tsx

```diff
@@ -95,6 +99,27 @@ export function BrowseView({ folderUID, width, height, canSelect }: BrowseViewPr
     [selectedItems, childrenByParentUID]
   );
 
+  if (status === 'pending') {
+    return <Spinner />;
+  }
+
+  if (status === 'fulfilled' && flatTree.length === 0) {
+    return (
+      <div style={{ width }}>
+        <EmptyListCTA
+          title={folderUID ? "This folder doesn't have any dashboards yet" : 'No dashboards yet. Create your first!'}
+          buttonIcon="plus"
+          buttonTitle="Create Dashboard"
+          buttonLink={folderUID ? `dashboard/new?folderUid=${folderUID}` : 'dashboard/new'}
+          proTip={folderUID && 'Add/move dashboards to your folder at ->'}
```

Comment:
I'd skip the arrow
```suggestion
          proTip={folderUID && 'Add/move dashboards to your folder at'}
```

Reply:
I agree - there's a bunch about these EmptyCTA components that I don't like, but I'm going to keep it the same as our current ones for now and we can revisit later.